samples/ios/Continote/Continote/Note.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Copyright (c) 2017 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import FirebaseDatabase
+
+/**
+ A Note for the user within Continote.
+ The schema of each Note is outlined in sample-database.rules.json within the web sample.
+
+ Each Note is considered immutable on the client-side to simplify syncing with the database for
+ this sample.
+ */
+struct Note {
+
+  // The title of this Note.
+  let title: String
+
+  // The main content of this Note.
+  let content: String
+
+  // The Firebase Realtime Database representation of this Note.
+  // This could be used to add or set a Note value in the database.
+  let asFirebaseData: [String : AnyObject]
```

Comment:
I actually had this as a computed property originally, but thought that wasn't necessary (and perhaps even confusing) since the value can never change after the initialization (since all of the properties it uses are also constants).

Could you clarify what you mean by the following, and perhaps how it addresses my concern above?:

> "In case the format changes, it makes sense to have all the encoding done here instead of allowing other places in the code to set it with their own representation. "

Thank you!